src/css/curCSS.js

```diff
@@ -9,7 +9,10 @@ function curCSS( elem, name, computed ) {
 
 	// getPropertyValue is needed for `.css('--customProperty')` (gh-3144)
 	if ( computed ) {
-		ret = computed.getPropertyValue( name ) || computed[ name ];
+		ret = computed.getPropertyValue( name );
+
+		// trim whitespace (issue #4926)
+		ret = ( ret && ret.trim() ) || computed[ name ];
```

Comment:
This trims all values. While this generally happens for regular properties, it seems safer to just trim where needed, i.e. for custom properties.

We have the `rcustomProp` regex defined in `css.js`; let's move it to a separate file `css/var/rcustomProp.js` (you can see other files in `var` for a pattern how it's used) and use both here and in `css.js` to detect custom properties and only trim in that case.

Reply:
> This trims all values. While this generally happens for regular properties, it seems safer to just trim where needed, i.e. for custom properties.

Yes, It's a good idea because the changes will be minimal. I will add restrictions.